### ISSUE TITLE
Namespace-wide annotation doesn't work

### ISSUE THREAD

**Issue by Woap** [bitnami-labs/sealed-secrets#169]
**Describe the bug** 
I want to use the annotation ( sealedsecrets.bitnami.com/namespace-wide: "true" ) but kubeseal doesn't seem to handle this annotation. 

**Expected behavior**
When using "namespace-wide" annotation, kubeseal has to give back a sealed secret with the corresponding annotation. 

With "cluster-wide" : 

Input : 
```yaml
apiVersion: v1
kind: Secret
metadata:
  annotations:
    sealedsecrets.bitnami.com/cluster-wide: "true"
  creationTimestamp: null
  name: secret
  namespace: dev
data:
     PASSWORD:  YWRtaW4K
     USER:  YWRtaW4K
```

Output : 
```yaml
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  annotations:
    sealedsecrets.bitnami.com/cluster-wide: "true"
  creationTimestamp: null
  name: secret
  namespace: dev
spec:
  encryptedData:
    PASSWORD: AgCQ5GcCkbzlxp8JLCf0azujTE2qz97W71qwgYcYtwaRkle6x19nnMo5c2bET/eryhJKqsIhudMvuNtG6Rkj28lTaJECq7a82SDLTz5oxbsQnKR6iqCxf3yuZaVTMJo54KKTuRIx9mcjTel8taE+EgKruoA46edEC/VdQpcUSfOJ4H+r+chB8IRmeQ9bp3ROB13Rpv37qzTAw7UuTbKjhSCzETrvOQWvcVYxKeJLgS7dankHKOIv6D6+UiVkow0QQDsNGMuJZxlIuXpYBU2Z9xC7uZdyDhPZzLvQmQX2ak1DXEQLoPp9KOla3WMFZEjxCOjd1IRMqRQCg4CfXqpGGUKdsNxXpBMdOWJ01NPqHwQcOEjaOfgysEnf1VNm3H3Smk7xjpxHLnVPoIKtdc9qYQ/0GOSevfjYNvoiGaFPc0+875UouVqv0BdE16ElXtvt/hal9k9dorbnrl1+so3m4fxYq9K+uMcDcC2pxclUJL/yJ5wMzXVzKKDCqHiub4a2Hh9g9c+64mCE5Ze1wElSGfonY7W+A3KyIuCH77URbAUlsoD3alNbsWKxPQLsxB/whzi5Ut+7NJ9DocesqWa9KQvLW4waBJYOytap5zAQo+QaMRKTxSQ1thjwG3hGVX4gUl7FNdPbbUZD+iRiIiEpl1HTAWTxn/+XGISuOKVSjhRU3wQtv/LWZiexku1UT4JAgv+3Og1YA30=
    USER: AgBV1g50yJ5JN8LxoFjLmnGYdI6OUTwLooAqID8d65sfYz4z+Aq/f64R3oUTKhHd0Ccepwb6i5BeOJz93zInLzuNQMBEAS1gC9Q51dqpqsufHIc/S2L6qJ5W79LRkCnhp0nY62bIe5nHiPmnKnRvYbFSzxtgdvB/tUa2zN++AQdLf7OHBT8fEWbt8fQRTp8psH8j5vtk45K03RRuj/eNuqzqHe55dch6Lmtm+y0y9YOHczf5WUR12mxRFO5FoExNNFOjKNvGXS5wL1E+E+fiAO21v1dzHWVkACGF52MI9RpjMMoz+Sd5rPwC2ugzIaqRvQqTByWgTGVuOMRcaE86qTqDd24L1ZrYdoJtYakJztF+uXG3GggmtHBX0+voWM2IBKj+YQT+inKoswHvb01eyCH4XtMKutpMjqflOHsmkOs54UFDck8PRaiKp7SYwseBapBH+I9M6sUzaXUN0UDsq8PnoTWYdequw6yNsLAT29fikn0861kuzUN0SjydzWOjf5mwwBSH1FSJXNxS+9b9RjzBmqJPcyntgT/0ZqsYjcitTQIMMGva2fwBnLoQajjCs21ndAo4a0KpJnN3BI+jC6DjlGk2lMewPMvUP3YOmfrupJTOuROCRwA1NZfiedVI4DQNRFYR9GzdyB/uXSBoSLPjvzxxT40MsOIJVUvjAcR+PkOzVLiD0f3budxjN2MZxJLPsIRIheI=
```

With "namespace-wide" :  

Input : 
```yaml
apiVersion: v1
kind: Secret
metadata:
  annotations:
    sealedsecrets.bitnami.com/namespace-wide: "true"
  creationTimestamp: null
  name: secret
  namespace: dev
data:
     PASSWORD:  YWRtaW4K
     USER:  YWRtaW4K
```

Output ( annotation namespace-wide is missing here ) :
```yaml
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  creationTimestamp: null
  name: secret
  namespace: dev
spec:
  encryptedData:
    PASSWORD: AgC2d1G4G6Nc5/xkyYDNSDJMQvLrumwgJQladaB6tMTSSXHwNPGqvgFwJMIvFHuWuoOZ/IB/rUB1rq39Zo6NeOrB4oKWyRki6yJ3M1CyxgviiOJhLr1twyMCEbfw7wG4Qk1bnKpxB9pBUdjk3C+xPmuMN42Fa9K2FxA9ixhoMOmcXAkaTVaM/1k4AOQp1Rp16TMDvkme6iyOU3GlEk5R+hlXDVZxhPMXGIXma7QepQnPYEKd2/rQKUO4XO0vvZ1QvAMQYNfaVkmKfLoG/dR8aO1CXRtSluZlSBK4mcx06pNtFBUbi6nALmUCb4zJnLjM2gArCuNEa3PBVxVzHWd92RQ4+BwG8rlGjoSe/u2s6RvlouIoOGeaT/7xzvQ0MlqegvRiFB0iiQcXb1ojYKDRKnO9UtQ1v/KW/K21mVhx6PZACTSL3k9vUF/7l23h5NXU4R2FWLj2+V+lKOCjZmI0w6+pui17bYUl0dSoMayOlCH2oaKyxcr3KKWX25apFVBG1U1t4DlwusfPKGunyGgkf0zgr7ZfOIsMVQg4FTSMUeF/drceMPk9mMqcs1WM0hyidrcE9SODtQgIA1pCpOcUDp12JwMTw9FNeI2ZqUT/mW47IWyY9H0dsBWi8snWowGNrxH6SjuAU4HC9XGsAtBYNchpSCLNyGH9lsE6pQqN0Wdy3sy/5aCJB5+/81JuHvn6ewB9q4ZTbpc=
    USER: AgASHX2mFq2JmIIQoQ4avDijThb7NcNdHB0I4Nfxqo6AR3PzfXhndcCD55ByJYpbUZJKYoik6eSt5zb4prJ8V2IN7F59X5TbDjGrAi6jSLRLwpS1V7eubdTeAWzPqcZLGyem/Rbqil5K8fH3IVDv37DvhEFpgZjjUJuTQc3fEavh7GS0xSJ91XYXXE8iwUhYds6/rFaj37Q1HDBKj1nD2ImwugXLuNl2oI+FWP+0cHyR+aopM0vhFi5czftgGvskGxQcLF8q72bI5W6PYSO+unquglcSOY7A7IOwzg4RrwkDm2nimHsjR/GowhRM0/jli3V/WQgTAgui4jLh6+A/s18KaL5ywRQ0da2P3qplQLuNXwk0sHYbrt3/8qnWycZxuVJ4PB5TlDvqZxompIE41iYNZZjDwpEMEuJjqTEQrE++pn1zLYCzQdg+g3V14PIiVuiB8Wy4wEd0QlWCjHG7VScVfj3ROBjM/ecfroxGEoi5RIAbHQlLkXjQ4lt6c3U0dl/u7V+41C/KDuz6bSOO4UuAjCVByZQLuSGxX2Dt1TTO2dcaCLmnL+J5sjcYIUSmHH2Xb+/+wxpvk536OaTQh5Q+C1WAyHd6nxhln19MMqtaE/1haB4TuxWJgcI5SSeSBRyjdIx0QLntG+F4SyCErXxAlg7isWCx6v42o3nqc2ROr/e1/F13HkfefKur9z0vHdWYQjXsh/Y=
```

So when I put this secret on Kube, the controller gives me an error ( updating dev/warped-rodent-croc-hunter-secret, will retry: crypto/rsa: decryption error) cause I've changed the name of the secret and it checks name/namespace equivalence between the SealedSecret and the Secret

**Comment by chrisharm** [comment]
There doesn't appear to be any support for the sealedsecrets.bitnami.com/namespace-wide annotation.

**Comment by chrisharm** [comment]
I should clarify my comment from yesterday.  The latest version on master appears to support this correctly.  

Using the latest kubeseal built from master, I seeing the correct behavior:

Input:
```yaml
apiVersion: v1
kind: Secret
metadata:
  annotations:
    sealedsecrets.bitnami.com/namespace-wide: "true"
  name: ss-ns-global
  namespace: default
data:
     PASSWORD:  YWRtaW4K
     USER:  YWRtaW4K
```

Output:
```yaml
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  annotations:
    sealedsecrets.bitnami.com/namespace-wide: "true"
  creationTimestamp: null
  name: ss-ns-global
  namespace: default
spec:
  encryptedData:
    PASSWORD: AgAMLXDv11LDa6b05V/SCtLVy4QRETBkmPcKt9MS4F8xTmzk36EmHf7C4+qLFCHwDaMANqWeGFwI8nTGlp4OfwpPQG5TIqi8WOVKSBlApfP5t+2KsEQsc4pUhdCGtIYOfSC9RIyBbMLs1hzwbPbMo/HYQsuH8CCdErSWbta03lalnVS3MEizE5p2WYV1H8lOpG/11KKKnDA/AjQn+gAViNRl54O9UTwnt0E+yGC4S7KUbyEHm4S7DlFlLPgoJOuoOLSC42kRgx7G9VYlO1sCKQ/Zl1ndjEST4ALvM/UliG5Lqs5dieVsDwlUzRnmd8RyI+OgabDP3kDhJcDs4vv2MVTAZpyYarkWkOVA2W1I24LGl2flYByFljt8MFz80o0LvFLy3HuDOsdig6du3B1/FnjedG7zvUgjsjlwY2eVetEn9jeKtLpujM0Baz9rzO5ZlXt2wUwt3Zqf7C+qbafnp9zWbylvCz41EQzU5A62CGbsGjKrfkGDH1PGhxp4FOAottLkzemStSesgVPMk6KgQ7p8LUPINn8zGFaCUhEGKUMXpMJ5t4ZyWEmQHUVQeNpTHuXjyR1lN8tQYVUkJBaYryjcmqBzfkgaKoPtsuS0Jj60pBRLhDMacvppIIVZ8Z02XXg0BvSPmW0HFf2Qq5iVNvEO20fzhcfgvnlrfqSFRE2EFaxtvsPMdRp69qiZV+tsjox12i7OPMg=
    USER: AgBs6rtFuDpQo4dRWjW+kYegwZw2pKynvLRB/C9adZJ8iihxLe3pBTvkoOmp42TtMKuhhb9qSdhdcvKb+4AcbOOjjmm5/G3GkMX+GBZ2ACh82dGRZLAA/evfGTvV/+3pP6IGCe6FaysGuHXC/efnFA3oU7vSyDdyZJ/KeQKf8YHfw//Uf/kDjZ2rSMgiZ3k8YkK3qbpcZfbz7YRmbiopThtqU3oqjOlhgQ3jY6qSY5nhhm9Xtyq0x0Y5LijHirqWsEMeWAeyZUwZ7rWD7NrVOtmE8Jva8QzD/04HGs+FFOGN/fNPZJmTvjflrnAg+1WE1JpJ4+7fiNAec3wNQ8T6oD93ZAPLr1kau8x1UunlgS2JDtuMudkYRKbWLE1G4xO6vGBpVz1hCQQRHogHYeUeGkQgRlSELyJTrRWwVnRv7iphNV4gIbbuD2zFT2+YOc1GU7Iph8VCP0ToPP3fkjY5JOk1QTL8po7jpVMTm3gm7j7z06rAGKYfKqqZeSDVLy0wRkm4z5hRNap65Jzu6dkNLRSbWNSNMxpqGRCvhF8Pq342PAJdHRjDgF1+PHs/RgbZjo+pQffVAG5hn1urmTyllCmWFf/H4HAKshUVDtI7/Ee85TACSNy+hybQ9yArIvYP/v9FLn0aoLgFsklyHKisP8fmiduV7wLw4lAKQWI3ZrUTR01hfNUTnSku6v+2fpWCsQCAvZvR7jg=
```

Like many of the other open issues in this project, this one is waiting on a new release.  You can follow along here #165 for details.

**Comment by Woap** [comment]
Resolved with last release (v0.8.0)
